test/TesterInternal/ActivationsLifeCycleTests/DeactivateOnIdleTests.cs

```diff
@@ -353,23 +361,35 @@ public void Dispose()
             }
 
             // Now we know that there's an activation; try both silos and deactivate it incorrectly
-            int primaryActivation = await testCluster.Primary.TestHook.UnregisterGrainForTesting(grainReference);
+            int primaryActivation =
+                await
+                    testCluster.Primary.GetTestHook(this.testCluster.InternalGrainFactory)
+                        .UnregisterGrainForTesting(grainReference);
             int secondaryActivation = 0;
 
             if (isMultipleSilosPresent)
             {
-                secondaryActivation = await testCluster.SecondarySilos[0].TestHook.UnregisterGrainForTesting(grainReference);
+                secondaryActivation =
+                    await
+                        testCluster.SecondarySilos[0].GetTestHook(this.testCluster.InternalGrainFactory)
```

Comment:
I'll add `GetTestHooks(SiloHandle)`  as an extension method in `OrleansTestingHost` for `IClusterClient`, so it will look like `client.GetTestHooks(silo)`